src/display/general/init.go

```diff
@@ -81,30 +88,89 @@ func NewCPUPage(numCores int) *CPUPage {
 
 // InitGeneral initializes all ui elements for the ui rendered by the grofer command
 func (page *MainPage) InitGeneral(numCores int) {
-
 	// Initialize Bar Graph for Memory Chart
-	page.MemoryChart.Title = " Memory (RAM) "
-	page.MemoryChart.TitleStyle = ui.NewStyle(ui.ColorClear)
-	page.MemoryChart.Labels = []string{"Total", "Available", "Used", "Free"}
-	page.MemoryChart.BarWidth = 8
-	page.MemoryChart.BarGap = 9
-	page.MemoryChart.BarColors = []ui.Color{ui.ColorCyan, ui.ColorGreen}
-	page.MemoryChart.LabelStyles = []ui.Style{ui.NewStyle(ui.ColorClear)}
-	page.MemoryChart.NumStyles = []ui.Style{ui.NewStyle(ui.ColorBlack)}
+	page.memoryChartWidget()
+	// Initialize Table for Disk Chart
+	page.diskChartWidget()
+	// Initialize Plot for Network Chart
+	page.networkChartWidget()
+	// Initialize Graph for CPU Usage
+	page.avgCpuGraphWidget()
+	if numCores > 8 || cpuTableVisible {
+		page.cpuTableWidget(numCores)
+	} else {
+		page.cpuGaugeWidget(numCores)
+	}
+	// Initialize Graph for Temperature Table
+	page.temperatureTableWidget()
```

Comment:
Maybe add a "init" as a prefix to these function names? They're quite vague as of now.

Reply:
hmm, interesting -- not part of this PR but should we consider [`init()`](https://golang.org/doc/effective_go#init) functions on a package level for things like this?

Reply:
Currently most of these "inits" are all for display widgets, so it might make sense, but then again, the inits are unique to each page which again is nested under the `display` package. We could look into it if we're considering a refactor.